app/models/user.rb

```diff
@@ -2,7 +2,7 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
```

Comment:
I was not able to login, signup or create new user to test API endpoints. I tried creating user via API endpoint, rails console and HTML form, all failed. 
![Screenshot 2021-12-25 at 18-29-42 Action Controller Exception caught](https://user-images.githubusercontent.com/1642014/147388465-1b5980c0-5f3b-4f53-8120-9007a188e97f.png)

![Screenshot from 2021-12-25 18-44-18](https://user-images.githubusercontent.com/1642014/147388632-068d34ed-6862-415e-8d4f-835466e35d82.png)